src/distributions/bernoulli.rs

```diff
@@ -68,6 +69,17 @@ pub enum BernoulliError {
     InvalidProbability,
 }
 
+impl fmt::Display for BernoulliError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str(match self {
+            BernoulliError::InvalidProbability => "p is outside 0..=1 in Bernoulli distribution",
```

Comment:
Sorry, but I'm not really a fan of using range syntaxes in documentation!

Here `0..=1` is used for floating-point types — but this is a Rust-specific expression and in Rust integers don't implicitly cast into floats. In the documentation we use the mathematical syntax `[0, 1]`; admittedly this is not universally recognised, but it does appear to be widely used — also, we should try to be consistent with our own documentation.